### DISCUSSION TITLE
Need help with Mimir/Loki/Tempo single-binary/distributed mode helm charts

### DISCUSSION THREAD

**Discussion by alita1991** [grafana/mimir#6211] [Help and support]
Hello,

I'm interested in exploring deployment possibilities for Mimir, Tempo, or Loki Helm charts that can address these situations:

- Kubernetes on a single node
- Kubernetes on multiple nodes

Could anyone please guide me to the correct helm charts that cover both scenarios? I currently explored the *-distributed charts, but while doing that, I found other charts for loki and tempo that run in single-binary mode, which is not available for mimir (or I was not able to find it, if there is one, please provide me a link).

Thanks

**Comment by 56quarters** [comment]
There's no monolithic mode (single-binary) for the Mimir helm chart and there's no intention to create one. If you're looking to quickly try out Mimir locally, there's a few [Docker compose](https://github.com/grafana/mimir/tree/main/development) setups you could use.

**Reply by alita1991** [reply]
Hi, thanks for your quick reply, regarding my question, is it okay to use the distributed chart version for Mimir/Loki/Tempo to cover single and multi-node k8s?
